userspace/libsinsp/eventformatter.cpp

```diff
@@ -280,6 +322,20 @@ sinsp_evt_formatter_cache::~sinsp_evt_formatter_cache()
 {
 }
 
+bool sinsp_evt_formatter_cache::resolve_tokens(sinsp_evt *evt, string &format, map<string,string>& values)
+{
+	auto it = m_formatter_cache.lower_bound(format);
+
+	if(it == m_formatter_cache.end() ||
```

Comment:
I would extract this piece of code in a separate `get_cached_formatter(format)` and use it both here and in `tostring`